app/views/courses/_statistics_course_info.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row">
+  <div class="col-md-8 page-heading">
+    <h1> <%= @course.name %> </h1>
+    <h3>
+      <%= "#{@course.group_name} группа" %>
+      <% if @teacher %>
+        <%= link_to edit_course_path(@course) do %>
+          <span class="glyphicon glyphicon-pencil"></span>
+        <% end %>
+      <% end %>
+    </h3>
+  </div>
+
+  <div class="col-md-4" align="right">
+    <h3> <%= @course.teacher.full_name %> </h3>
+    <h3>
+      <% if @teacher || @student %>
```

Comment:
Looks like a duplication of the `_course_info.html.erb`. Why do you need this code twice?